docs/tool/js/views/filter-view.js

```diff
@@ -714,6 +706,188 @@ var filterView = {
         setState('nullsShown.' + c.source, ths.nullsShown);
 
     },
+    zoneAggregateFilterControl: function(component){
```

Comment:
I'm concerned about the quantity of duplicated code in this approach. Is it necessary to create a whole new FilterControl element? 

I think the core difference between this FilterControl and the normal ContinuousFilterControl is that the data column that needs to be used depends on which zone type is currently selected. The min/max then will also need to change. Can we put those couple of changes into their own function or if statement and then use the existing code for the rest of it?